plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorSmokeTest.java

```diff
@@ -15,22 +15,52 @@
 
 import com.google.common.collect.ImmutableMap;
 import io.trino.testing.QueryRunner;
+import org.testng.annotations.AfterClass;
 
 import static io.trino.plugin.clickhouse.ClickHouseQueryRunner.createClickHouseQueryRunner;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestClickHouseConnectorSmokeTest
         extends BaseClickHouseConnectorSmokeTest
 {
+    protected TestingClickHouseServer clickHouseServer;
+
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
+        clickHouseServer = new TestingClickHouseServer();
```

Comment:
```suggestion
        clickHouseServer = closeAfterClass(new TestingClickHouseServer());
```

That will allow you to remove the `destroy` method.